### PR TITLE
Feat: Added manta rules

### PR DESCRIPTION
**Pull request by marrip**
Adding T/N manta run

### CODE REVIEW

**Review by jonca79 — COMMENTED** [review]
There is no temp() on the output files.

**Comment by marrip** [comment]
I added `temp()` on the `.vcf.gz` files which are unnecessary but I would like to keep the somatic results.

File: workflow/rules/config_manta_tn.smk [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+rule config_manta_tn:
+    input:
+        bam_t="alignment/merge_bam/{sample}_T.bam",
+        bai_t="alignment/merge_bam/{sample}_T.bam.bai",
+        bam_n="alignment/merge_bam/{sample}_N.bam",
+        bai_n="alignment/merge_bam/{sample}_N.bam.bai",
+        ref=config["reference"]["fasta"],
+    output:
+        scrpt="cnv_sv/manta/{sample}/runWorkflow.py",
```
Comment (Smeds) [inline workflow/rules/config_manta_tn.smk:15]: should this be a temp output?

File: workflow/rules/manta_run_workflow_tn.smk [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        ref=config["reference"]["fasta"],
+        scrpt="cnv_sv/manta/{sample}/runWorkflow.py",
+    output:
+        cand_si_vcf=temp("cnv_sv/manta/{sample}/results/variants/candidateSmallIndels.vcf.gz"),
+        cand_si_tbi=temp("cnv_sv/manta/{sample}/results/variants/candidateSmallIndels.vcf.gz.tbi"),
+        cand_sv_vcf=temp("cnv_sv/manta/{sample}/results/variants/candidateSV.vcf.gz"),
+        cand_sv_tbi=temp("cnv_sv/manta/{sample}/results/variants/candidateSV.vcf.gz.tbi"),
+        dipl_sv_vcf=temp("cnv_sv/manta/{sample}/results/variants/diploidSV.vcf.gz"),
+        dipl_sv_tbi=temp("cnv_sv/manta/{sample}/results/variants/diploidSV.vcf.gz.tbi"),
+        som_sv_vcf="cnv_sv/manta/{sample}/results/variants/somaticSV.vcf.gz",
```
Comment (Smeds) [inline workflow/rules/manta_run_workflow_tn.smk:22]: we should write down something about when temp should be applied to output values, in most existing rule I believe we have it on all output.

File: workflow/rules/config_manta_tn.smk (hunk 1 above)
Comment (marrip) [reply workflow/rules/config_manta_tn.smk]: done

File: workflow/rules/manta_run_workflow_tn.smk (hunk 2 above)
Comment (marrip) [reply workflow/rules/manta_run_workflow_tn.smk:22]: Ah ok, might be a good idea to minimize the footprint of our workflows.

**Review by Smeds — APPROVED** [review]

**Pull request merged**